client/src/components/pages/admins/ViewAllAdmins.js

```diff
@@ -15,7 +15,8 @@ export class ViewAllAdmins extends Component {
 
     componentDidMount() {
         // call redux action to retrieve all admins from api
-        this.props.adminActions.filter();
+        this.props.adminActions.filter({ accessLevel: { $ne: 'Volunteer' } });
```

Comment:
remove these changes I already fixed this, I changed it in the action definition instead